src/altinn-app-frontend/src/components/base/RadioButtons/RadioButtonsContainerComponent.tsx

```diff
@@ -1,17 +1,13 @@
 import React from 'react';
 import type { IComponentProps } from '../../index';
-import type { IMapping, IOption, IOptionSource } from 'src/types';
 import { useRadioButtons } from 'src/components/base/RadioButtons/radioButtonsUtils';
 import { ControlledRadioGroup } from 'src/components/base/RadioButtons/ControlledRadioGroup';
+import type { ILayoutCompRadioButtons } from 'src/features/form/layout';
 
-export interface IRadioButtonsContainerProps extends IComponentProps {
+export interface IRadioButtonsContainerProps
+  extends IComponentProps,
+    ILayoutCompRadioButtons {
   validationMessages?: any;
```

Comment:
Why is `validationMessages` not also moved to `ILayoutCompRadioButtons` ?

Reply:
I'm not exactly sure where that value is supposed to come from, but it's neither `IComponentProps` (properties from `GenericComponent` when rendered from there) nor `ILayoutCompRadioButtons` (properties defined for that component in the layout file). There were a few cases like this. The component apparently expects to sometimes get a value with that property name, but I'm not sure exactly where it is supposed to come from - other than it surely is some internal state that might not belong to either of the two interfaces it inherits from.